packages/react-native-sdk/README.md

```diff
@@ -1,30 +1,23 @@
 # Kinvey React Native SDK
 
-## Installation
+The Kinvey React Native SDK is used to develop an React Native application that connects to Kinvey.
 
-From the command prompt go to your app's root folder and execute:
+## Install
+
+Using npm:
 
 ```bash
 npm i kinvey-react-native-sdk
 ```
 
-## Basic Usage
-
-### Demo app
-
-## Notes
```

Comment:
Need to maintain a section in the RN README for now that mentions the fact that this is not a fully-supported shim yet, so that developers are aware.

Reply:
This is fixed with 2aaa1bf.